docs[ProblemStatementAndGoals] Section 1.2 inputs & outputs

chenans1 opened the pull request:
Rough draft of inputs and outputs for the problem statement and goals

chenans1 committed:
docs[ProblemStatementAndGoals] Section 1.2 inputs & outputs

Rough draft of inputs and outputs for the problem statement and goals

bahsounm reviewed (commented):
- Overall i think it looks good, but I would suggest some more explanation for some things
- what do we mean by pseudo-music generator, maybe expand more on that, explain the pseudo part so its more clear for someone who has no idea what going on
- Also in some parts it sounds like 3rd person, and in others its not that it sounds first person but rather not formal enough, so id try to make it sounds more formal (this may include things like "the system will" rather than "it will/there will" things like that)
- Im not sure how detailed we have to be with our points for this doc but something like this "for system 1, it should output a list of recommended tracks and some form of a similarity score". what do we mean by "some form of a similarity score"? Im not sure if its supposed to be abstract or not, if it is cool, you good, but if not then we can maybe add more detail, even if it not what we are actually going to do, make it something suggestive. ie "for system 1, it should output a list of recommended tracks and some form of a similarity score, <then give a potential example of what that is and what it indicates>"
-

chenans1 committed:
docs[ProblemStatementAndGoals] updated appendix, reflection section 2.

Not sure about writing how we dealt with conflict resolution on latex. It seems kind of too obvious and duh to be written about but I'm honestly out of ideas.

AhmedAl-Hayali committed:
Merge branch 'main' into Project-Desc-Input-Outputs
